ast/cinaps/grammar.mli

```diff
@@ -1,5 +1,9 @@
-val string_of_ty : internal: bool -> Astlib.Grammar.ty -> string
+val string_of_targ : ?nodify: unit -> internal: bool -> Astlib.Grammar.targ -> string
+
+val string_of_ty : ?nodify: unit -> internal: bool -> Astlib.Grammar.ty -> string
```

Comment:
Let's document what `?nodify` does and why we need it. It was not obvious on my first read, I had to go digging around.